docs/ldml/tr35-dates.md

```diff
@@ -1068,40 +1068,62 @@ As in other cases, **narrow** may be ambiguous out of context.
 <!ATTLIST era aliases NMTOKENS #IMPLIED >
 ```
 
-The `<calendarData>` element now provides only locale-independent data about calendar behaviors via its `<calendar>` subelements, which for each calendar can specify the astronomical basis of the calendar (solar, lunar, etc.) and the date ranges for its eras.
-
-Era start or end dates are specified in terms of the equivalent proleptic Gregorian date (in "y-M-d" format). Eras may be open-ended, with unspecified start or end dates. For example, here are the eras for the Gregorian calendar:
+The `<calendarData>` element now provides only locale-independent data about calendar behaviors via its `<calendar>` subelements,
+which for each calendar can specify the astronomical basis of the calendar (solar, lunar, etc.) and the date ranges for its eras.
+For example:
 
 ```xml
-<era type="0" end="0-12-31" code="gregory-inverse" aliases="bc bce"/>
-<era type="1" start="1-01-01" code="gregory" aliases="ad ce"/>
+<calendar type="gregorian">
+  <calendarSystem type="solar" />
+  <eras>
+    <era type="0" end="0-12-31" code="bce" aliases="bc"/> <!-- Before Common Era, Before Christ -->
+    <era type="1" start="1-01-01" code="ce" aliases="ad"/> <!-- Common Era, Anno Domini -->
+  </eras>
+</calendar>
 ```
+The `type` has an integer value. 
+The type values do not have to start at 0, nor be contiguous.
+However, they must be unique for the calendar, and define the ordering of `era`s within that calendar.
```

Comment:
this sounds like it defines a chronological ordering, which it does not

Reply:
> IMO, we should deprecate the end attribute, and say that all of the eras must have a start date (except optionally the first).

That makes sense. Currently only inverse eras have an end date, but there it's always the day before the next start, so it's kind of redundant.

> Now, if it is already the case that all but possibly one era has a start date, we could move to that this release.

That's the case.

> That way the ordering is firmly established by the ordering of the start dates, and the ordering of the types is irrelevant.

That's the ordering I also have in mind, but here you say that the type defines an ordering, which is wrong.

Reply:
I redid https://unicode-org.atlassian.net/browse/CLDR-19023 to make it clearer, and will bring this up in the Design meeting today to see if we can fast-track it.

Reply:
@robertbastian please let me know if there are any problems you see in that.

Reply:
lg, just the timezone thing I already commented on